### ISSUE TITLE
the computation of contrastive loss

### ISSUE THREAD

**Issue by lonelylingoes** [Simon4Yan/Learning-via-Translation#7]
In "Learning-via-Translation\SPGAN\models\models_spgan.py", why the computation of contrastive loss divide the number of "3.0", like this "m_loss = tf.identity(( T_metric_POS + S_metric_POS + 2*NEG)/3.0 , name = 'metric_losses')", is that should be "4.0"?
